package.json

```diff
@@ -0,0 +1,38 @@
+{
+  "name": "leaderboard",
+  "version": "1.0.0",
+  "description": "The leaderboard website displays scores submitted by different players. It also allows you to submit your score. All data is preserved thanks to the external Leaderboard API service.",
+  "main": "index.js",
+  "scripts": {
+    "test": "jest --coverage",
+    "build": "webpack",
+    "start": "webpack serve --open"
+  },
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/iammouaz/leaderboard.git"
+  },
+  "keywords": [],
+  "author": "",
+  "license": "ISC",
+  "bugs": {
+    "url": "https://github.com/iammouaz/leaderboard/issues"
+  },
+  "homepage": "https://github.com/iammouaz/leaderboard#readme",
+  "devDependencies": {
```

Comment:
- [ ] OPTIONAL
Please install _webpack-dev-serve_ package so as to enable your application to be run on the server